packages/console/src/pages/SignInExperience/tabs/SignUpAndSignInTab.tsx

```diff
@@ -22,10 +22,10 @@ const SignInMethodsTab = ({ defaultData, isDataDirty }: Props) => {
 
   return (
     <>
-      <SignInMethodsForm />
+      <SignUpAndSignInForm />
```

Comment:
Can we seperate this into several forms, as we did before, every section(have a title in Figma) has a form.

Reply:
We do this as we did before.
Just replace the `SignInMethodsForm` with the new one, and they are equivalent in structure.

Reply:
Reorged.